Godeps/Godeps.json

```diff
@@ -406,6 +406,11 @@
 			"Rev": "a1b8ba5163b7f041b22761461eabd02b70d1f824"
 		},
 		{
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Comment": "v0.2-17-gc18eea6",
```

Comment:
The last tagged version is quite old, so I asked for a new one: https://github.com/gogo/protobuf/issues/182